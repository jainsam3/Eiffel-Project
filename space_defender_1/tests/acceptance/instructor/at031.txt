--Firing multiple time with different projectile movement. 
play(5,5,1,1)
fire
fire
fire
fire
fire
fire
abort
play(5,5,1,2)
fire
fire
fire
abort
play(5,5,1,3)
fire
fire
fire
abort
play(5,10,1,3)
fire
fire
fire
fire
abort
play(5,10,1,4)
fire
fire
fire
fire
abort
play(5,10,1,5)
fire
fire
fire
abort

