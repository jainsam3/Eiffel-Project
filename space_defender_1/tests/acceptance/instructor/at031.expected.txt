  Welcome to Space Defender Version 1.
->play(5,5,1,1)
  state:1.0, ok
      1  2  3  4  5
    A _  _  _  _  _
    B _  _  _  _  _
    C S  _  _  _  _
    D _  _  _  _  _
    E _  _  _  _  _
->fire
  state:2.0, ok
  The Starfighter fires a projectile at: [C,1]
      1  2  3  4  5
    A _  _  _  _  _
    B _  _  _  _  _
    C S  *  _  _  _
    D _  _  _  _  _
    E _  _  _  _  _
->fire
  state:3.0, ok
  A projectile moves: [C,2] -> [C,3]
  The Starfighter fires a projectile at: [C,1]
      1  2  3  4  5
    A _  _  _  _  _
    B _  _  _  _  _
    C S  *  *  _  _
    D _  _  _  _  _
    E _  _  _  _  _
->fire
  state:4.0, ok
  A projectile moves: [C,3] -> [C,4]
  A projectile moves: [C,2] -> [C,3]
  The Starfighter fires a projectile at: [C,1]
      1  2  3  4  5
    A _  _  _  _  _
    B _  _  _  _  _
    C S  *  *  *  _
    D _  _  _  _  _
    E _  _  _  _  _
->fire
  state:5.0, ok
  A projectile moves: [C,4] -> [C,5]
  A projectile moves: [C,3] -> [C,4]
  A projectile moves: [C,2] -> [C,3]
  The Starfighter fires a projectile at: [C,1]
      1  2  3  4  5
    A _  _  _  _  _
    B _  _  _  _  _
    C S  *  *  *  *
    D _  _  _  _  _
    E _  _  _  _  _
->fire
  state:6.0, ok
  A projectile moves: [C,5] -> out of the board
  A projectile moves: [C,4] -> [C,5]
  A projectile moves: [C,3] -> [C,4]
  A projectile moves: [C,2] -> [C,3]
  The Starfighter fires a projectile at: [C,1]
      1  2  3  4  5
    A _  _  _  _  _
    B _  _  _  _  _
    C S  *  *  *  *
    D _  _  _  _  _
    E _  _  _  _  _
->fire
  state:7.0, ok
  A projectile moves: [C,5] -> out of the board
  A projectile moves: [C,4] -> [C,5]
  A projectile moves: [C,3] -> [C,4]
  A projectile moves: [C,2] -> [C,3]
  The Starfighter fires a projectile at: [C,1]
      1  2  3  4  5
    A _  _  _  _  _
    B _  _  _  _  _
    C S  *  *  *  *
    D _  _  _  _  _
    E _  _  _  _  _
->abort
  state:8.0, ok
  Game has been exited.
->play(5,5,1,2)
  state:9.0, ok
      1  2  3  4  5
    A _  _  _  _  _
    B _  _  _  _  _
    C S  _  _  _  _
    D _  _  _  _  _
    E _  _  _  _  _
->fire
  state:10.0, ok
  The Starfighter fires a projectile at: [C,1]
      1  2  3  4  5
    A _  _  _  _  _
    B _  _  _  _  _
    C S  *  _  _  _
    D _  _  _  _  _
    E _  _  _  _  _
->fire
  state:11.0, ok
  A projectile moves: [C,2] -> [C,4]
  The Starfighter fires a projectile at: [C,1]
      1  2  3  4  5
    A _  _  _  _  _
    B _  _  _  _  _
    C S  *  _  *  _
    D _  _  _  _  _
    E _  _  _  _  _
->fire
  state:12.0, ok
  A projectile moves: [C,4] -> out of the board
  A projectile moves: [C,2] -> [C,4]
  The Starfighter fires a projectile at: [C,1]
      1  2  3  4  5
    A _  _  _  _  _
    B _  _  _  _  _
    C S  *  _  *  _
    D _  _  _  _  _
    E _  _  _  _  _
->abort
  state:13.0, ok
  Game has been exited.
->play(5,5,1,3)
  state:14.0, ok
      1  2  3  4  5
    A _  _  _  _  _
    B _  _  _  _  _
    C S  _  _  _  _
    D _  _  _  _  _
    E _  _  _  _  _
->fire
  state:15.0, ok
  The Starfighter fires a projectile at: [C,1]
      1  2  3  4  5
    A _  _  _  _  _
    B _  _  _  _  _
    C S  *  _  _  _
    D _  _  _  _  _
    E _  _  _  _  _
->fire
  state:16.0, ok
  A projectile moves: [C,2] -> [C,5]
  The Starfighter fires a projectile at: [C,1]
      1  2  3  4  5
    A _  _  _  _  _
    B _  _  _  _  _
    C S  *  _  _  *
    D _  _  _  _  _
    E _  _  _  _  _
->fire
  state:17.0, ok
  A projectile moves: [C,5] -> out of the board
  A projectile moves: [C,2] -> [C,5]
  The Starfighter fires a projectile at: [C,1]
      1  2  3  4  5
    A _  _  _  _  _
    B _  _  _  _  _
    C S  *  _  _  *
    D _  _  _  _  _
    E _  _  _  _  _
->abort
  state:18.0, ok
  Game has been exited.
->play(5,10,1,3)
  state:19.0, ok
      1  2  3  4  5  6  7  8  9 10
    A _  _  _  _  _  _  _  _  _  _
    B _  _  _  _  _  _  _  _  _  _
    C S  _  _  _  _  _  _  _  _  _
    D _  _  _  _  _  _  _  _  _  _
    E _  _  _  _  _  _  _  _  _  _
->fire
  state:20.0, ok
  The Starfighter fires a projectile at: [C,1]
      1  2  3  4  5  6  7  8  9 10
    A _  _  _  _  _  _  _  _  _  _
    B _  _  _  _  _  _  _  _  _  _
    C S  *  _  _  _  _  _  _  _  _
    D _  _  _  _  _  _  _  _  _  _
    E _  _  _  _  _  _  _  _  _  _
->fire
  state:21.0, ok
  A projectile moves: [C,2] -> [C,5]
  The Starfighter fires a projectile at: [C,1]
      1  2  3  4  5  6  7  8  9 10
    A _  _  _  _  _  _  _  _  _  _
    B _  _  _  _  _  _  _  _  _  _
    C S  *  _  _  *  _  _  _  _  _
    D _  _  _  _  _  _  _  _  _  _
    E _  _  _  _  _  _  _  _  _  _
->fire
  state:22.0, ok
  A projectile moves: [C,5] -> [C,8]
  A projectile moves: [C,2] -> [C,5]
  The Starfighter fires a projectile at: [C,1]
      1  2  3  4  5  6  7  8  9 10
    A _  _  _  _  _  _  _  _  _  _
    B _  _  _  _  _  _  _  _  _  _
    C S  *  _  _  *  _  _  *  _  _
    D _  _  _  _  _  _  _  _  _  _
    E _  _  _  _  _  _  _  _  _  _
->fire
  state:23.0, ok
  A projectile moves: [C,8] -> out of the board
  A projectile moves: [C,5] -> [C,8]
  A projectile moves: [C,2] -> [C,5]
  The Starfighter fires a projectile at: [C,1]
      1  2  3  4  5  6  7  8  9 10
    A _  _  _  _  _  _  _  _  _  _
    B _  _  _  _  _  _  _  _  _  _
    C S  *  _  _  *  _  _  *  _  _
    D _  _  _  _  _  _  _  _  _  _
    E _  _  _  _  _  _  _  _  _  _
->abort
  state:24.0, ok
  Game has been exited.
->play(5,10,1,4)
  state:25.0, ok
      1  2  3  4  5  6  7  8  9 10
    A _  _  _  _  _  _  _  _  _  _
    B _  _  _  _  _  _  _  _  _  _
    C S  _  _  _  _  _  _  _  _  _
    D _  _  _  _  _  _  _  _  _  _
    E _  _  _  _  _  _  _  _  _  _
->fire
  state:26.0, ok
  The Starfighter fires a projectile at: [C,1]
      1  2  3  4  5  6  7  8  9 10
    A _  _  _  _  _  _  _  _  _  _
    B _  _  _  _  _  _  _  _  _  _
    C S  *  _  _  _  _  _  _  _  _
    D _  _  _  _  _  _  _  _  _  _
    E _  _  _  _  _  _  _  _  _  _
->fire
  state:27.0, ok
  A projectile moves: [C,2] -> [C,6]
  The Starfighter fires a projectile at: [C,1]
      1  2  3  4  5  6  7  8  9 10
    A _  _  _  _  _  _  _  _  _  _
    B _  _  _  _  _  _  _  _  _  _
    C S  *  _  _  _  *  _  _  _  _
    D _  _  _  _  _  _  _  _  _  _
    E _  _  _  _  _  _  _  _  _  _
->fire
  state:28.0, ok
  A projectile moves: [C,6] -> [C,10]
  A projectile moves: [C,2] -> [C,6]
  The Starfighter fires a projectile at: [C,1]
      1  2  3  4  5  6  7  8  9 10
    A _  _  _  _  _  _  _  _  _  _
    B _  _  _  _  _  _  _  _  _  _
    C S  *  _  _  _  *  _  _  _  *
    D _  _  _  _  _  _  _  _  _  _
    E _  _  _  _  _  _  _  _  _  _
->fire
  state:29.0, ok
  A projectile moves: [C,10] -> out of the board
  A projectile moves: [C,6] -> [C,10]
  A projectile moves: [C,2] -> [C,6]
  The Starfighter fires a projectile at: [C,1]
      1  2  3  4  5  6  7  8  9 10
    A _  _  _  _  _  _  _  _  _  _
    B _  _  _  _  _  _  _  _  _  _
    C S  *  _  _  _  *  _  _  _  *
    D _  _  _  _  _  _  _  _  _  _
    E _  _  _  _  _  _  _  _  _  _
->abort
  state:30.0, ok
  Game has been exited.
->play(5,10,1,5)
  state:31.0, ok
      1  2  3  4  5  6  7  8  9 10
    A _  _  _  _  _  _  _  _  _  _
    B _  _  _  _  _  _  _  _  _  _
    C S  _  _  _  _  _  _  _  _  _
    D _  _  _  _  _  _  _  _  _  _
    E _  _  _  _  _  _  _  _  _  _
->fire
  state:32.0, ok
  The Starfighter fires a projectile at: [C,1]
      1  2  3  4  5  6  7  8  9 10
    A _  _  _  _  _  _  _  _  _  _
    B _  _  _  _  _  _  _  _  _  _
    C S  *  _  _  _  _  _  _  _  _
    D _  _  _  _  _  _  _  _  _  _
    E _  _  _  _  _  _  _  _  _  _
->fire
  state:33.0, ok
  A projectile moves: [C,2] -> [C,7]
  The Starfighter fires a projectile at: [C,1]
      1  2  3  4  5  6  7  8  9 10
    A _  _  _  _  _  _  _  _  _  _
    B _  _  _  _  _  _  _  _  _  _
    C S  *  _  _  _  _  *  _  _  _
    D _  _  _  _  _  _  _  _  _  _
    E _  _  _  _  _  _  _  _  _  _
->fire
  state:34.0, ok
  A projectile moves: [C,7] -> out of the board
  A projectile moves: [C,2] -> [C,7]
  The Starfighter fires a projectile at: [C,1]
      1  2  3  4  5  6  7  8  9 10
    A _  _  _  _  _  _  _  _  _  _
    B _  _  _  _  _  _  _  _  _  _
    C S  *  _  _  _  _  *  _  _  _
    D _  _  _  _  _  _  _  _  _  _
    E _  _  _  _  _  _  _  _  _  _
->abort
  state:35.0, ok
  Game has been exited.
